docker-compose.yml

```
@@ -1,5 +1,11 @@
 version: '3.8'
 services:
+  redis:
+    container_name: sce-redis-dev
+    image: redis:latest
+    restart: always
+    volumes:
+      - sce_redis_data_dev:/data/redis
```

make `sce_redis_data` a volume at the bottom of the file
```suggestion
      - sce_redis_data:/data/redis
```

---

done